pkg/api/types.go

```diff
@@ -221,6 +224,24 @@ type GCEPersistentDiskVolumeSource struct {
 	ReadOnly bool `json:"readOnly,omitempty"`
 }
 
+// A ISCSI Disk can only be mounted as read/write once.
+type ISCSIDiskVolumeSource struct {
+	// Required: iSCSI target IP
+	TargetIP string `json:"targetIP,omitempty"`
```

Comment:
I know I commented on "portal" before - what I meant to convey was that if "portal" was just a random word, please change it, but if it actually had meaning in iSCSI land, then it was OK.  You said it had meaning, but then changed it.

BUT...

You call it `targetIP` - it's noit an IP, it's an IP:port pair.  And it's not obvious that it could not be hostname:port - so the field name and comment are wrong.

Does it have to be a monolithic field or could it be two fields (Host, Port)?  I don't want to make it not be idiomatic for iSCSI users, but that's how we break down HTTP addresses, for example.